src/raglite/_extract.py

```diff
@@ -52,6 +52,7 @@ class MyNameResponse(BaseModel):
                 "name": return_type.__name__,
                 "description": return_type.__doc__ or "",
                 "schema": return_type.model_json_schema(),
+                "strict": True,
```

Comment:
We cannot enable strict mode for all extraction unfortunately: it severely limits the types of Pydantic schemas you can extract data from. Instead, what I would suggest is that you introduce an optional `strict: bool = False` keyword argument to the extract function (with a default of `False` so we support all schemas by default), and then to set `strict=True` for the eval use case where we can guarantee it will work.